http_service/bugbug_http/models.py

```diff
@@ -32,41 +30,34 @@
     "testlabelselect",
     "testgroupselect",
 ]
+
 DEFAULT_EXPIRATION_TTL = 7 * 24 * 3600  # A week
+redis = Redis.from_url(os.environ.get("REDIS_URL", "redis://localhost/0"))
 
 
-MODEL_LAST_LOADED: Dict[str, datetime] = {}
-MODEL_CACHE: Dict[str, Model] = {}
+def load_model_for_service(model_name):
+    LOGGER.info("Recreating the %r model in cache" % model_name)
+    try:
+        return load_model(model_name)
+    except FileNotFoundError:
+        if ALLOW_MISSING_MODELS:
+            LOGGER.info(
+                "Missing %r model, skipping because ALLOW_MISSING_MODELS is set"
+                % model_name
+            )
+            return None
+        else:
+            raise
 
 
-redis = Redis.from_url(os.environ.get("REDIS_URL", "redis://localhost/0"))
+MODEL_CACHE: ReadthroughTTLCache[str, Model] = ReadthroughTTLCache(
+    timedelta(hours=2), load_model_for_service
+)
+MODEL_CACHE.start_ttl_thread()
```

Comment:
Nit: maybe we could just start the thread from the ReadthroughTTLCache constructor. I'll let you decide, I don't have a strong preference

Reply:
I think its better to keep it separate for the sake of our unit tests. 

We need nonzero values for the TTL to unit test this object, but if we try to unit test using a real thread rather that the datetime mocking I was doing, the tests will noticeably delay unit test runs, and also will open the door for non deterministic behavior since I dont think theres actually a guarantee that the sleep function sleeps for the amount of time it says its going to sleep for.

Reply:
on second thought, it might be possible to unit test despite this, but it'd be a bit tricky if it is

Reply:
Feel free to keep it as-is, I don't mind :)